components/tip.js

```diff
@@ -0,0 +1,36 @@
+// node_modules
+import { Popover } from "@headlessui/react";
+import { useState } from "react";
+import { usePopper } from "react-popper";
+import PropTypes from "prop-types";
+
+/**
+ * When the children element is clicked a panel comes up on
+ * top of the page with the given text content.
+ */
+export default function Tooltip({ content, children }) {
```

Comment:
Since the default function is called `Tooltip`, could you name the file `tooltip.js`? Makes it easier to predict one from the other when using this.